common/cmd_bootm.c

```diff
@@ -159,6 +159,7 @@ extern void lynxkdi_boot( image_header_t * );
 
 image_header_t header;
 ulong load_addr =  CFG_LOAD_ADDR;		/* Default Load Address */
+ulong sys_bootflag =  0x5A5A5A5A;		/* Default Load Address */
```

Comment:
better to have a definition elsewhere: ex #defined BOOT_FROM_A 0x5A5A5A5A

Reply:
@truongn1 You are absolutely right, but I didn't have enough time for a perfect design. It was too rushed